README.md

```diff
@@ -112,6 +114,8 @@ To install this library via [npm](https://www.npmjs.org) **version 3**, use the
 npm install @reactivex/rxjs
 ```
 
+This will include all builds for CJS/ES6/AMD/TypeScript.
```

Comment:
@kwonoj does README actually need to mention also `npm install rxjs`? Doesn't `npm install @reactivex/rxjs` work for every bundling system? You can either use CommonJS, or ES6 and transpile it, or AMD, or TypeScript and compile it with either es5/es6 module option? Maybe `npm install rxjs` can be removed?